apps/chat/src/utils/creditPeriods.ts

```
@@ -0,0 +1,168 @@
+      return new Date(
+        Date.UTC(
+          startOfWeek.getUTCFullYear(),
+          startOfWeek.getUTCMonth(),
+          startOfWeek.getUTCDate()
+        )
+      )
+
+    case ResetPeriod.BIWEEKLY:
+      // Start of current biweekly period (every other Monday)
+      // Use a reference date to ensure consistent biweekly periods
+      const referenceDate = new Date('2025-09-15T00:00:00.000Z') // Semester Start 2025
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)